doc/whats_new.rst

```diff
@@ -139,6 +139,8 @@ API
 
     - Added :class:`mne.decoding.UnsupervisedSpatialFilter` providing interface for scikit-learn decomposition algorithms to be used with MNE data, by `Jean-Remi King`_ and `Asish Panda`_
 
+    - class:`mne.decoding.FilterEstimator` has been deprecated in favor of :class:`mne.decoding.Filterer` by `Asish Panda`_
```

Comment:
wasn't there a bug in `FilterEstimator` that @kingjr mentioned? Could you also fix that? Thanks!